### DISCUSSION TITLE
Does xmpp-web support BOSH protocol?

### DISCUSSION THREAD

**Discussion by VA1DER** [nioc/xmpp-web#105] [Q&A] [answered]
A readme for a Prosody BOSH plugin suggests that xmpp-web supports connecting via the BOSH protocol.  Was/is that the case?

**Comment by nioc** [comment, marked as answer]
Hello, yes it was before the [0.9.0](https://github.com/nioc/xmpp-web/releases/tag/0.9.0) version shipping #45.

Moving to xmpp.js remove this feature for now.
